tests/robot-tests/README.md

```diff
@@ -59,137 +58,139 @@ If you intend to run the tests from your local machine, you will also need to cr
+- `-e` - This option dictates what environment to test against. You will need a corresponding `.env` file to match the environment you're testing against (i.e. a `.env.dev` if you are testing against development etc.)
 
-If you need to change the chromedriver version used by the CI pipeline, it can be done in `scripts/pipeline-run-rf-tests.sh`. You can check [this repository](https://github.com/actions/virtual-environments/tree/master/images) for the version of chrome used on the Azure agent you're using. At the time of writing, the robot tests use the [Ubuntu 1604 image](https://github.com/actions/virtual-environments/blob/master/images/linux/Ubuntu1604-README.md)
+- `-f` - Path to test file or folder. If you choose a folder it will run all robot tests in that folder
 
+- `-v` - Dictates whether Chrome runs in visual or headless mode. 
 
-## How do I backup and restore the test data on my local environment?
+- `--timeout` - Specifies robot timeout in second (default is 30)
 
-By following the instructions in this section, you can backup and restore all the data from your local environment, including content and statistics databases and all important Azure storage blobs and table.
+- `--implicit-wait` - Specifices implicit wait in seconds (default is 5)
 
-You can currently only backup and restore data on your local environment on Windows. This is because currently you can only emulate Azure storage tables on Windows. At the time of writing [Azurite V3 only has support for blobs and queues, not tables](https://github.com/Azure/Azurite).
+## **Running tests on the pipeline**
 
-For the backup and restore scripts to work, you'll need:
+After changing the tests and ensuring they pass locally, it is then recommended that you run your changes on the pipeline to ensure they also pass there. There is a build pipeline you can use to check any changes you've made will pass on the pipeline: 'UI Tests'. You can run this pipeline using the branch with your changes, and it will produce an artifact you can download with the test report, screenshots, etc. In addition to this you can also run the UI tests locally targeting development if you do not have Azure access.
 
-- to be running the MsSQL database. You can do this using a docker container -- as per `explore-education-statistics/src/docker-compose.yml` -- from the src directory, `docker-compose up db`
-- to be running AzureStorageEmulator
-- to have AzCopy v7.3 installed (`C:\Program Files (x86)\Microsoft SDKs\Azure\AzCopy\AzCopy.exe` -- you can change where in the backup and restore scripts if you've installed it elsewhere)
-- optionally, you might want Azure Data Studio and Azure Storage Explorer to inspect the MsSQL databases and your emulated blob, queue, and table storage.
+At the time of writing, the pipeline runs the tests against the Dev environment, and runs all tests. You can edit the pipeline if you wish to do something different.
```

Comment:
All the tests that are not marked "NotAgainstDev" did you mean, or currently are there no tests marked with "NotAgainstDev"?